index.js

```diff
@@ -5,14 +5,46 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "pink"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+ console.log('This is what the traveler object looks like: ', traveler)
+let travelerInfo = document.querySelector('.traveler')
+let travelerName = travelerInfo.querySelector('h2')
+let travelerNickname = travelerInfo.querySelector('em')
+let travelerPhoto = travelerInfo.querySelector('img')
+travelerName.textContent = `${traveler.name}`
+travelerNickname.textContent = `${traveler.nickname}`
+travelerPhoto.src = `${traveler.photo}`
+travelerPhoto.alt = "Cairn terrier on wood floor"
```

Comment:
Good! You don't necessarily need to use template strings here though because you're not interpolating any strings together. Doesn't make much of a difference though!

```suggestion
travelerName.textContent = traveler.name
travelerNickname.textContent = traveler.nickname
travelerPhoto.src = traveler.photo
travelerPhoto.alt = "Cairn terrier on wood floor"
```